frontend/appflowy_flutter/lib/workspace/presentation/home/menu/view/view_item.dart

```diff
@@ -254,7 +256,16 @@ class _SingleInnerViewItemState extends State<SingleInnerViewItem> {
 
     return GestureDetector(
       behavior: HitTestBehavior.translucent,
-      onTap: () => widget.onSelected(widget.view),
+      onTap: () {
+        if (RawKeyboard.instance.isControlPressed) {
+          context.read<TabsBloc>().openTab(widget.view);
+        }
+        widget.onSelected(widget.view);
```

Comment:
Move this logic to `personal_folder.dart` and `favorite_folder.dart`. 

For example, here's the `onSelected` implementation in the personal folder.

```dart
                    onSelected: (view) {
                      getIt<TabsBloc>().add(
                        TabsEvent.openPlugin(
                          plugin: view.plugin(),
                          view: view,
                        ),
                      );
                    },
```